x-pack/plugins/apm/server/lib/helpers/get_document_sources.ts

```diff
@@ -17,58 +16,127 @@ export async function getDocumentSources({
   start,
   end,
   kuery,
+  enableServiceTransactionMetrics,
```

Comment:
Perhaps add a comment to the top of this function describing why we are doing this

Reply:
should it be here or in the ui setting?